downstream/modules/platform/proc-controller-user-permissions.adoc

```diff
@@ -1,38 +1,22 @@
+:_mod-docs-content-type: PROCEDURE
+
 [id="proc-controller-user-permissions"]
 
-= Adding and removing user permissions
+= Adding roles to a team
 
-To add permissions to a particular user:
+You can assign permissions to teams, such as edit and administer resources and other elements.
+You can set permissions through an inventory, project, job template and other resources, or within the Organizations view.
 
 .Procedure
-. From the *Users* list view, click on the name of a user.
-. On the *Details* page, click btn:[Add]. 
-This opens the *Add user permissions* wizard.
-+
-image:users-add-permissions-form.png[Add Permissions Form]
-. Select the object to a assign permissions, for which the user will have access.
-. Click btn:[Next].
-. Select the resource to assign team roles and click btn:[Next].
+. From the navigation panel, select {MenuAMTeams}.
+. Select the team *Name* to which you want to add roles.
+. Select the *Roles* tab and click btn:[Add roles].
 +
-image:users-permissions-IG-select.png[image]
-
-. Select the resource you want to assign permissions to. 
-Different resources have different options available.
+include::snippets/snip-gw-roles-note-multiple-components.adoc
 +
-image:users-permissions-IG-roles.png[image]
-
-. Click btn:[Save].
-. The *Roles* page displays the updated profile for the user with the permissions assigned for each selected resource.
-
-[NOTE]
-====
-You can also add teams, individual, or multiple users and assign them permissions at the object level.
-This includes templates, credentials, inventories, projects, organizations, or instance groups. 
-This feature reduces the time for an organization to onboard many users at one time.
-====
-
-.To remove permissions:
-* Click the image:disassociate.png[Disassociate,10,10] icon next to the resource. 
-This launches a confirmation dialog asking you to confirm the disassociation.
-
-
+. Select a *Resource type* and click btn:[Next]. 
+. Select the resources that will receive new roles and click btn:[Next].
```

Comment:
Select the resource s to receive the new roles...

Reply:
Fixed.